examples/ex1.cpp

```diff
@@ -134,48 +141,63 @@ int main(int argc, char *argv[])
    b->AddDomainIntegrator(new DomainLFIntegrator(one));
    b->Assemble();
 
-   // 7. Define the solution vector x as a finite element grid function
+   // 7. Set MFEM config parameters from the command line options
+   config::usePA(pa);
+   if (pa) { mesh->EnsureNodes(); }
```

Comment:
I like the `EnsureNodes()` method - very good idea to finally have an easy way to ensure we have a gridfunction for the nodes!